scripts/linters/pylint_extensions.py

```diff
@@ -540,11 +595,60 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node,
             accept_no_param_doc=node_allow_no_param)
 
-    def check_arg_section_indentation(self, node):
-        """Checks whether the function argument definitions ("Args": section)
-        are indented properly. Parameters should be indented by 4 relative to
-        the 'Arg: ' line and any wrap-around descriptions should be indented
-        by 8. The rest of the docstring section checks are in DocstringChecker.
+    def check_docstring_section_style(self, node):
+        """Checks whether the docstring has valid style.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        if node.name in self.constructor_names:
+            class_node = checker_utils.node_frame_class(node)
+            if class_node is not None:
+                self.check_docstring_structure(class_node)
+        self.check_docstring_structure(node)
+
+    def check_docstring_structure(self, node):
+        """Checks whether the docstring has the correct structure i.e.
+        do not have space the beginning and have a period at the end of
+        docstring.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        if node.doc:
+            docstring = node.doc.splitlines()
+            # Check for space after """ in docstring.
+            if docstring[0][0] == b' ':
+                self.add_message('space-after-triple-quote', node=node)
+            # Check if single line docstring span two lines.
+            elif len(docstring) == 2 and docstring[-1].strip() == b'':
```

Comment:
Why is this converted to `elif`? It should be `if`, right?

Reply:
Changed to if